### PR TITLE
fix: parseENSAddress #1200

### PR DESCRIPTION
**Pull request by gkucmierz**

### CODE REVIEW

**Comment by gkucmierz** [comment]
Screenshot 2020-11-07 at 07 40 29

It is possible to add tokens list with dashes in domain name

**Comment by gkucmierz** [comment]
How to run linter locally?

`npm run lint` is not defined

File: src/utils/parseENSAddress.ts [hunk 1]
```diff
@@ -1,7 +1,7 @@
-const ENS_NAME_REGEX = /^(([a-zA-Z0-9]+\.)+)eth(\/.*)?$/
+const ENS_NAME_REGEX = /^(([a-zA-Z0-9]([a-zA-Z0-9-]+[a-zA-Z0-9])\.)+)eth(\/.*)?$/
```
Comment (moodysalem) [inline src/utils/parseENSAddress.ts:1]: does this work with 1-2 character ens names?

Comment (gkucmierz) [reply src/utils/parseENSAddress.ts:1]: Nope, let me fix it

**Comment by gkucmierz** [comment]
@moodysalem can you check it once again?

**Review by moodysalem — APPROVED** [review]

**Pull request merged**